### DISCUSSION TITLE
LBB + PI Juice + MSata Hard Drive Board and a Touch Screen

### DISCUSSION THREAD

**Discussion by erhard2019** [outdoorbits/little-backup-box#269] [Ideas]
Hi All, I am new to all this with Linux and raspberrypi,  but I saw this LBB project and I will be traveling later this year and my wife and I like photography and want to backup photos and video while traveling. I was thinking this would work pretty good, but was hoping to expand on it. I'd like to use an msata hard drive (vs thumbdrives), Pi 4, Pi juice (for power) and a touch screen to see what is going on (but willing to change that out). 

I wasn't sure that this could work with LBB?

Any help or feedback would be greatly appreciated....

Hardware

Geekworm X850 V3.1 mSATA SSD Storage Expansion Board USB3.0 Mobile Hard Disk Module Compatible with Raspberry Pi 3 Model B+/Pi 3 Model B

Raspberry Pi 5 Motherboard

PiJuice Portable Power Platform for Raspberry Pi

**Comment by outdoorbits** [comment]
The power supply by Pi juice might not be a problem. msata is not enabled yet as I don't own one. But I do not expect to run into trouble. If you are able to execute a few simple commands to give me necessary information I hope it will be easy to establish that. Of course, I would need you to test this. But maybe it's not the worst having usb connected backups redundant.
The only doubt I have is the touchscreen. If you can disable the auto mount of the graphic surface this should work using a browser. But for sure, you'll always have a better display using your smartphone - and you save a lot of battery power.

**Reply by outdoorbits** [reply]
If you are about to buy all these components, I would think about using raspberry pi 5. On every board before, the usb power supply caused problems in lots of times. These are easy to handle by a powered usb hub but power supply is the pi 5 is stronger if you use an adequate power source. If you like to use the image viewer you definitively have an advantage by the faster pi 4 or 5. Over that, pi 4 and 5 have WiFi integrated.

**Comment by erhard2019** [comment]
OK, yeah, I'll change thr Pi to a Pi4 and I'll get ride of the touch screen.
And with the MSata board it has a USB connection.
We take a lot of photos while we travel, and that is why I was thinking of using the MSata board/drive for storage.
Also the PiJuice for stable power to the Pi

Thank you for the feedback!

**Comment by outdoorbits** [comment]
Good news for you. If you still want to use a touchscreen, just have a look into the wiki

**Comment by deadesq** [comment]
As @outdoorbits mentioned, there's a wiki on the touchscreen.  That's what I set up, and it works great.  I'm using tiny USB thumb drives as the backup target, just to save some size, but the touchscreen setup works great.  I'm going to be building one in a couple of weeks, and I'm going to do a YouTube walkthrough video.

**Comment by outdoorbits** [comment]
I'm very excited to see this!
